app/features-json/repos_json_controller.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "../shared/authenticated_controller_base"
+require_relative "./view_models/repo_view_model"
+
+module FastlaneCI
+  # Controller for providing all data relating to projects
+  class RepositoryJSONController < AuthenticatedControllerBase
+    HOME = "/data/repos"
+
+    get HOME.to_s do
```

Comment:
For my own knowledge, is this best practice for Ruby or `"#{HOME}"`?

Reply:
Rubocop: prefer `.to_s` over string interpolation 😆